src/utils/dock-helper.js

```diff
@@ -0,0 +1,16 @@
+const getDock = ({ dock, chart, rtl }) => {
```

Comment:
We do have the dockService already... is this something that could get extracted from there maybe?

Reply:
This dock is legend dock. Normal data legend get calculated dock position when set to auto from color-service. 
DockService seems is axes dock?
![Screenshot 2021-11-08 at 13 48 27](https://user-images.githubusercontent.com/25456307/140745010-c116dd2f-c0a0-4e53-81d7-2d78a872d05f.png)

Reply:
Allright... I see we have some similar utility file - `layout-utils.js` in la-vie... that is something we might want to move into qlik-chart-modules at some point, and then this could belong in there maybe. But for now, let's go with this.